.github/workflows/push-images.yml

```diff
@@ -0,0 +1,49 @@
+name: Push-Images
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  push-images:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+        
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+          
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Gather Docker Meta Data
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            ghcr.io/mlflow/mlflow
+          # generate Docker tags based on the following events/attributes
+          tags: |
+            type=ref,event=tag
+      
+      - name: Build and Push Base Image
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
```

Comment:
Curious what `${{ steps.meta.outputs.tags }}` looks like.

Reply:
```
JSON output
  {
    "tags": [
      "ghcr.io/oojo12/mlflow:1.26.0",
      "ghcr.io/oojo12/mlflow:latest"
    ],
    "labels": {
      "org.opencontainers.image.title": "mlflow",
      "org.opencontainers.image.description": "Open source platform for the machine learning lifecycle",
      "org.opencontainers.image.url": "https://github.com/oojo12/mlflow",
      "org.opencontainers.image.source": "https://github.com/oojo12/mlflow",
      "org.opencontainers.image.version": "1.26.0",
      "org.opencontainers.image.created": "2022-09-10T16:37:39.884Z",
      "org.opencontainers.image.revision": "8d5e6c247[45](https://github.com/oojo12/mlflow/runs/8286027551?check_suite_focus=true#step:6:53)7386[48](https://github.com/oojo12/mlflow/runs/8286027551?check_suite_focus=true#step:6:56)24c86882408018d9f62bc25",
      "org.opencontainers.image.licenses": "Apache-2.0"
    }
  }















```

Reply:
the output can be seen in the Gather meta data dn build + push images steps [here](https://github.com/oojo12/mlflow/runs/8286027551?check_suite_focus=true)

Reply:
Thanks!